src/routeConfiguration.js

```diff
@@ -108,34 +108,39 @@ const routeConfiguration = () => {
       component: () => (
         <NamedRedirect
           name="EditListingPage"
-          params={{ slug: draftSlug, id: draftId, type: 'new', tab: 'description' }}
+          params={{ slug: draftSlug, id: draftId, listingType: 'sauna', type: 'new', tab: config.firstSaunaTab }}
         />
       ),
     },
+
     {
       path: '/l-equipment/new',
```

Comment:
forgot to change this 'l-equipment' to just 'l'. this has been fixed by the very next commit